src/test/java/io/knotx/junit5/assertions/KnotxAssertionsTest.java

```diff
@@ -41,4 +50,169 @@ void stringsThatDifferNewlineAreEqual() {
     final String expected = "a b c";
     assertEqualsIgnoreWhitespace(actual, expected);
   }
+
+  // ****************************
+  // assertJsonEquals
+  // ****************************
```

Comment:
More scenarios
- two json objects with multiple (2 or 3) keys added with different sequence (just to make sure, that order of key does not mess the method logic)
- how about tests for `assertJsonArrayEquals`, looks like the order of elements is important (should it be?)

Reply:
> two json objects with multiple (2 or 3) keys added with different sequence (just to make sure, that order of key does not mess the method logic)
added

> how about tests for assertJsonArrayEquals, looks like the order of elements is important (should it be?)
it should, test added

`assertJsonArrayEquals` method is private, we can expose it when required.